cmd/root.go

```diff
@@ -49,7 +49,7 @@ var rootCmd = &cobra.Command{
 		echo := echo.New()
 		echo.HideBanner = true
 		echo.Use(middleware.Logger())
-		echo.Use(core.DecodeURIPath)
+		//echo.Use(core.DecodeURIPath)
```

Comment:
I'd say just remove it when we don't need it anymore

Reply:
done